libdrgn/arch_s390_defs.py

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+REGISTERS = [
+    *[DrgnRegister(f"r{i}") for i in range(0,16)],
```

Comment:
This is equivalent to the preferred form without the start index:

```suggestion
    *[DrgnRegister(f"r{i}") for i in range(16)],
```